content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -405,7 +406,7 @@ If you have pods that are impacted from the default 1 second timeout,
 you should update their probe timeout so that you're ready for the
 eventual removal of that feature gate.
 
-With the fix of the defect, for exec probes, on Kubernetes `1.20+` with the `dockershim` container runtime,
+With the fix of the defect, for exec probes, on Kubernetes `1.20+` with the `containerd` container runtime,
 the process inside the container may keep running even after probe returned failure because of the timeout.
```

Comment:
I'm not sure about this change. I don't know that the issue is present when you use containerd as the container runtime. I recommit omitting this update from the current PR, and then opening an new PR (for the _dev-1.24_ branch) that is specifically about this paragraph.

Reply:
1. I'll have to update this PR unaltering line 408
2. A new PR (for the dev-1.24 branch) specific to this issue to be opened? 
is that what you meant @sftim?

Reply:
@SubhasmitaSw yup, unalter 408 and track it in a new PR or issue that targets that branch :)

Reply:
@shannonxtreme Thank you, I'll open a separate pr.